scheduler/prioritize.go

```diff
@@ -29,16 +30,40 @@ func capacityToScore(capacity uint64, divisor float64) int {
 	}
 }
 
-func scoreNodes(nodes []corev1.Node, divisor float64) []HostPriority {
+func scoreNodes(pod *corev1.Pod, nodes []corev1.Node, defaultDivisor float64, divisors map[string]float64) []HostPriority {
 	result := make([]HostPriority, len(nodes))
 
+	var dcs []string
+	for k := range pod.Annotations {
+		if strings.HasPrefix(k, topolvm.CapacityKeyPrefix) {
+			dcs = append(dcs, k[len(topolvm.CapacityKeyPrefix):])
+		}
+	}
+	if len(dcs) == 0 {
+		return nil
+	}
+
 	for i, item := range nodes {
```

Comment:
The same applies here; please make the inside a separate function and call it with a goroutine.